packages/bitcore-node/test/unit/models/wallet.unit.ts

```diff
@@ -1,7 +1,58 @@
-import {expect} from 'chai';
+import { expect } from 'chai';
+import * as sinon from 'sinon';
+import { WalletModel, IWalletModel } from '../../../src/models/wallet';
+import { WalletAddressModel } from '../../../src/models/walletAddress';
 
-describe('Wallet Model', function(){
-    it('should have a test which runs', function(){
-        expect(true).to.equal(true);
+describe('Wallet Model', function () {
+
+  describe('_apiTransform', () => {
+    it('should return the transform object with wallet info', () => {
+      let wallet = {
+        name: 'Wallet1',
+        singleAddress: true,
+        pubKey: 'xpub661MyMwAqRbcFa63vSTa3vmRiVWbpLWhgUsyvjfMFP7ePR5osC1rtPUkgJrB94V1YEQathfWLm9U5zaZttYPDPWhASwJGUvYvPGtofqnTGN',
+        path: 'm/44\'/0\'/0\''
+      }
+
+      const result = WalletModel._apiTransform(new WalletModel(wallet), {
+        object: false
+      });
+      const parseResult = JSON.parse(result);
+
+      expect(parseResult.name).to.be.equal(wallet.name);
+      expect(parseResult.pubKey).to.be.equal(wallet.pubKey);
+
+    });
+  });
+
+  describe('updateCoins', () => {
+    let sandbox;
+    beforeEach(() => {
+      sandbox = sinon.sandbox.create();
     });
-});
\ No newline at end of file
+    afterEach(() => {
+      sandbox.restore();
+    });
+    it('should return wallet address model update coins', async () => {
```

Comment:
I'd change this name to say that it calls `WalletAddressModel.updateCoins` instead of saying it returns.  Since you are really just checking to see that it is called

Reply:
resolved